### PR TITLE
Added Ocata support, removed Liberty support and helpers sync

### PR DESCRIPTION
**Pull request by ionutbalutoiu**
* Added Ocata support
* Removed Liberty support
* Synced ADCharmUtils and OpenStackCommon

### CODE REVIEW

**Comment by gabriel-samfira** [comment]
```${env:COMPUTERNAME}``` is all caps on Nano, that is why we switched to ```[System.Net.Dns]::GetHostName()```. This has the potential to confuse schedulers, that care about caps.

File: lib/Modules/CinderHooks/CinderHooks.psm1 [hunk 1]
```diff
@@ -356,6 +322,8 @@ function New-CharmServices {
                         -DisplayName $charmServices[$key]["display_name"] `
                         -Description $charmServices[$key]["description"] `
                         -Confirm:$false
+            Start-ExternalCommand { sc.exe failure $charmServices[$key]["service"] reset=5 actions=restart/1000 }
```
Comment (gabriel-samfira) [inline lib/Modules/CinderHooks/CinderHooks.psm1:325]: does this fail if run multiple times? do we need some idem-potency here?

File: lib/Modules/CinderHooks/CinderHooks.psm1 [hunk 2]
```diff
@@ -537,6 +482,8 @@ function Invoke-ConfigChangedHook {
                     $status = (Get-Service -Name $svc).Status
                     if($status -eq [System.ServiceProcess.ServiceControllerStatus]::Running) {
                         Restart-Service -Name $svc
+                    } else {
+                        Stop-Service -Name $svc
```
Comment (gabriel-samfira) [inline lib/Modules/CinderHooks/CinderHooks.psm1]: I don't understand why we're stopping here. If the service is running, we restart, but if it's not running we stop it?

**Review by gabriel-samfira — CHANGES_REQUESTED** [review]

File: lib/Modules/CinderHooks/CinderHooks.psm1 (hunk 1 above)
Comment (ionutbalutoiu) [reply lib/Modules/CinderHooks/CinderHooks.psm1:325]: It doesn't fail. All good and idem-potent.

File: lib/Modules/CinderHooks/CinderHooks.psm1 (hunk 2 above)
Comment (ionutbalutoiu) [reply lib/Modules/CinderHooks/CinderHooks.psm1]: Tired mind then, I guess. I will remove it. :) Makes no sense.

**Pull request merged**